app/models/review.rb

```diff
@@ -0,0 +1,7 @@
+class Review < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :video
+
+  validates_presence_of :rating, :body
+  #validates :rating, inclusion: { in: (1.0..5.0), message: "rating must be from 1 to 5" }
```

Comment:
This is a tiny nitpick... unless you plan on using this in the future you can delete it.